base/abstractarray.jl

```diff
@@ -3340,6 +3344,8 @@ end
 Like [`map`](@ref), but stores the result in `destination` rather than a new
 collection. `destination` must be at least as large as the smallest collection.
 
+The order in which the `collection`s are iterated is undefined. If `destination` shares memory with any of the `collection`s, the result might be incorrect.
```

Comment:
why is this one "might be incorrect" vs the others' "is undefined" ?

also, since `collection...` is a vararg, I think (to be cautious) that it is slightly ambiguous whether this language refers to (the order of iteration of the elements of each/any constituent collection), or (the ordering of the constituent collections themselves) or both

Reply:
apologies to be adding comments so late in the editing process

Reply:
just clarified that